terraform/gcp/security/gcp-org-impersonation-roles-iam-binding.yaml

```diff
@@ -0,0 +1,23 @@
+rules:
+    - id: gcp-org-impersonation-roles-iam-binding
+      patterns:
+        - pattern: resource
+        - pattern-inside: |
+              resource "google_organization_iam_binding" "..." {
+              ...
+              role = "roles/editor"
+              members = [ ... ]

```

Comment:
Based on the message written, it seems like the issue this rule tries to find is where `role = "roles/editor" and one of the members is a service account. Is this is case, or is the only thing that's important to find the fact that `roles/editor` is used?

Reply:
Yes, the issue is with the editor role (https://cloud.google.com/iam/docs/understanding-roles).